File: komisi_salesman2.py
print('Masukkan nama: ', end='')
nama = input()

print('Masukkan nilai penjualan: ', end='')
nilaiPenjualan = int(input())

komisi = int(0.05 * float(nilaiPenjualan))

print()
print('Komisi salesman "' + nama + '": ' + str(komisi))

File: KomisiSalesman2.java
import java.util.Scanner;

public class KomisiSalesman2 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.print("Masukkan nama: ");
        String nama = in.nextLine();

        System.out.print("Masukkan nilai penjualan: ");
        int nilaiPenjualan = in.nextInt();

        int komisi = (int) (0.05 * (double) nilaiPenjualan);

        System.out.println();
        System.out.println("Komisi salesman \"" + nama + "\": " + komisi);

        in.close();
    }
}
